packages/types/Credential.d.ts

```diff
@@ -30,6 +32,8 @@ export type CredentialForCalendarServiceWithEmail = CredentialPayload & {
       client_id: string;
       private_key: string;
     };
+    id?: string;
+    key: Prisma.JsonValue | null;
```

Comment:
Why do we have differnet types for key in `CredentialForCalendarServiceWithEmail` and `CredentialForCalendarService`?. I think they were meant to have just one difference that one has `client_email` required and other one do not. So, lets try to keep it same.